AutowareAuto/src/perception/filters/point_cloud_fusion_nodes/param/point_cloud_fusion.param.yaml

```diff
@@ -0,0 +1,7 @@
+
+---
+/**:
+  ros__parameters:
+    number_of_sources: 2
+    output_frame_id:  "concat_velodyne"
+    cloud_size:       80000
```

Comment:
I noticed `80000` was used instead of the `55000` in the original param file, just curious how this now value was determined?

Reply:
I found that each of the Velodynes published point clouds with `35,000 +/- 1,000` points when echoing messages on the truck. I added a fairly generous buffer of `10,000` points to make sure we don't drop messages that contain more points than expected, but this number could definitely be brought down if desired